engine-rest/engine-rest-openapi/src/main/templates/paths/metrics/get.ftl

```diff
@@ -26,7 +29,9 @@
                 "history-cleanup-removed-case-instances",
                 "history-cleanup-removed-decision-instances",
                 "history-cleanup-removed-batch-operations",
-                "history-cleanup-removed-task-metrics"]
+                "history-cleanup-removed-task-metrics",
+                "process-instances",
+                "task-users"]
```

Comment:
Just a late hint for reference: we don't support task user metrics in the interval endpoint here, that's why `unique-task-users` wasn't present.